CMakeLists.txt

```diff
@@ -88,7 +114,18 @@ target_link_libraries(autoapp
                         ${WINSOCK2_LIBRARIES}
                         ${RTAUDIO_LIBRARIES}
                         ${AASDK_PROTO_LIBRARIES}
-                        ${AASDK_LIBRARIES})
+                        ${AASDK_LIBRARIES}
+                        ${QTGLIB_LIBRARY}
+                        ${QTGLIB_LIBRARIES}
+                        ${QTGSTREAMER_LIBRARY}
+                        ${QTGSTREAMER_LIBRARIES}
+                        ${QTGSTREAMER_UI_LIBRARY}
+                        ${QTGSTREAMER_UI_LIBRARIES}
+                        ${QTGSTREAMER_QUICK_LIBRARIES}
+                        ${QTGSTREAMER_QUICK_LIBRARY}
+                        ${GOBJECT_LIBRARIES}
+)
```

Comment:
this should be at the end of `${GOBJECT_LIBRARIES}` 

and remove the extra newline after